community/gst-plugins/depends

```diff
@@ -0,0 +1,22 @@
+alsa-lib
+ffmpeg
+gdk-pixbuf
+gstreamer
+gst-plugins-base
+lame
+lcms2
+libXdamage
+libjpeg-turbo
+libpng
+libressl
+libsoup
+libtheora
```

Comment:
I'm thinking of dropping this lib from the repositories as I've never actually seen media which uses it.

Reply:
It also hasn't seen a release in 9 years from a quick google. If it can reasonably be dropped from the repositories it'd be nice from a security perspective.